azure-pipelines/end-to-end-tests-dir/add.ps1

```diff
@@ -9,18 +9,77 @@ New-Item -Path $manifestDir -ItemType Directory
 New-Item -Path $manifestPath -ItemType File `
     -Value (ConvertTo-Json -Depth 5 -InputObject $vcpkgJson)
 
-Run-Vcpkg add port zlib @manifestDirArgs
+Run-Vcpkg add port sqlite3 @manifestDirArgs
 Throw-IfFailed
 
 $expected = @"
 {
   "dependencies": [
-    "zlib"
+    "sqlite3"
   ]
 }
 "@
 
 $actual = (Get-Content -Path $manifestPath -Raw).TrimEnd()
 if ($expected -ne $actual) {
-    throw "Add port didn't add zlib dependency correctly.`nExpected: $expected`nActual:$actual"
+    throw "Add port didn't add sqlite3 dependency correctly.`nExpected: $expected`nActual:$actual"
+}
+
+# Add "default-features": false in comparison to the previous test
+Run-Vcpkg add port "sqlite3[core]" @manifestDirArgs
+Throw-IfFailed
+$expected = @"
+{
+  "dependencies": [
+    {
+      "name": "sqlite3",
+      "default-features": false
```

Comment:
I disagree with this behavior: `add port` should never *reduce* that which gets installed.

Reply:
I have fixed this now